docs/UserGuide.md

```diff
@@ -291,10 +291,10 @@ Format:
 addskill INDEX [s/SKILL]...
 ```
 
-* Only valid indexes are edited. <br>
-  e.g If there are only `4` applicants in the app but `5` is specified, then the intended action will not be carried out.
-* Only positive indexes are edited. <br>
-  e.g As we label our applicants incrementally starting from `1` (e.g `1, 2, 3, and so on`), an index of `-1` will not be tagged to an applicant.
+* Index of applicant specified must not be more than the total number of applicants in LinkedOUT. <br>
```

Comment:
This does give more insight than the previous statement, but is there any way to simplify this? It seems like a very 'CS' way of thinking. `this must not violate cond1`. If I can think of a better alternative I will let you know.

Reply:
Not sure if there's a more layman but clearer way to put this